static/js/marker.js

```diff
@@ -299,4 +299,26 @@ var MarkerView = Backbone.View.extend({
             }
         }
     },
+    select : function(){
+        //unselect all previous selections
+        _.each(app.markerList, function(markerView){
+            markerView.unselect();
+        });
+        //if marker is currently shown
+        this.marker.setTitle(this.marker.getTitle() + "(נבחר)");
+    },
+    unselect : function(){
+        //if was selected and currently displayed as single icon
+        if(this.isSelected()){
+            this.marker.setTitle(this.marker.getTitle().replace("(נבחר)", ""));
+        }
+    },
+    isSelected : function(){
+        //if shown and selected
+        if(this.marker.getTitle().search("(נבחר)") != -1){
+            return true;
```

Comment:
This construct makes no sense: the construct `if (...) { return true; } else { return false; }` should be replaced by `return (...)`.